prow/getting_started.md

```diff
@@ -141,10 +141,16 @@ plugins:
 Replace `YOUR_ORG/YOUR_REPO:` with the appropriate values. If you want, you can
 instead just say `YOUR_ORG:` and the plugin will run for every repo in the org.
 
-Run the following to test the file, replacing the path as necessary:
+Next, create an empty file called `config.yaml`:
```

Comment:
Instead of requiring `--config-path` to be specified (necessitating the empty `config.yaml`) we could fix `check-config` to only validate `plugins.yaml` if that is all that is provided. This comment is correct now so lets keep it, but we might want to change that behavior in another PR.